docs/en/observability/exploratory-data-visualizations.asciidoc

```diff
@@ -1,28 +1,201 @@
+To drilldown into a specific time period, click and drag a selection then *Apply
+changes*.
 
-include::synthetic-monitoring-visualizations.asciidoc[leveloffset=+1]
+To add visualizations to an existing case, click *Add to case* from the toolbar.
 
-include::user-experience-visualizations.asciidoc[leveloffset=+1]
+To customize a visualization further, click *Open in Lens* from the toolbar to
+modify visualizations with the drag and drop editor. To learn more, see
+{kibana-ref}/lens.html[Lens].
 
+[discrete]
+[[report-types]]
+== What report types are available?
 
+[options,header]
+|===
+|Report type | Description
+
+|KPI over time
+|The KPI over time histogram represents the performance indicators based on the
+metric you select, such as page views or monitor duration.
+
+|Performance distribution
+|The Performance distribution time-series chart enables you to examine the
+perceived performance of your web applications based on the metric you select.
+
+|Core web vitals
+|The Core web vitals chart is a graphical representation of key metrics, such as
+loading performance, load responsiveness, and visual stability, for each of your
+web applications.
+
+To learn more about metrics such as Largest contentful paint, First input delay,
+and Cumulative layout shift, see {observability-guide}/user-experience.html#user-experience-metrics[{user-experience} metrics].
+
+|Device distribution
+|The Device distribution chart displays device information such as OS, carrier name, and connection type.
+
+|===
+
+For a breakdown of which data types are available for which reports, see <<data-types>>
+
+You can create multi-series visualizations for each report type but you cannot
+combine different report types in one visualization. To change report type for
+your visualization, click *Remove series* and then *Add series*.
+
+[discrete]
+[[create-multi-series-visualizations]]
+== Create multi-series visualizations
+
+The Explore data view is currently enabled for the following apps:
+
+* Metrics
+* Uptime
+* User Experience
+
+To create a multi-series visualization:
+
+. Click *Explore data* from a compatible app.
+* The report type will default to the most appropriate for the app but you can
+edit the series or add more series to the visualization. For example, if you
+click *Explore Data* from the Uptime app, the `Synthetics monitoring`
+ data type and `Monitor duration` report metric will be preselected.
+. Click *Add series* to define an additional series for the visualization.
+. Click *Select data type* and choose from the following options:
+* Synthetics monitoring
+* User experience (RUM)
+* Infra metrics
+* Mobile experience
+. Click *Select report metric* and select the options and filters you need. You
+will see a *Missing...* warning if required fields (highlighted with red
+  underline) are incomplete.
+. Click *Apply changes* to see the updated visualization or repeat the *Add series* process to expand the visualization.
```

Comment:
```suggestion
. Click *Apply changes* to see the updated visualization, or repeat the *Add series* process to expand the visualization.
```

Reply:
Typically I wouldn't suggest using a comma between a subject and verb, but here the "you" is implied and it helps readability.